shopfloor/services/cluster_picking.py

```diff
@@ -583,6 +637,13 @@ def scan_destination_pack(self, picking_batch_id, move_line_id, barcode, quantit
         have the same destination.
         * start_line: to pick the next line if any.
         """
+        if self.work.menu.no_prefill_qty:
+            res = self.scan_line(picking_batch_id, move_line_id, barcode)
+            # If we got an error from scan line, fall back to finding
+            # the destination package
+            message_type = res.get("message", {}).get("message_type")
+            if message_type != "error":
+                return res
```

Comment:
I see no relation to this change and the specs ?
Commenting this code does not make the tests fail ?

Reply:
That is because the test are all based on `scan_line`